src/util/analytics-client.ts

```diff
@@ -10,11 +10,10 @@ const appKey = `com.github.integration${instance ? `.${instance}` : ""}`;
 
 let analyticsNodeClient;
 
-export function sendAnalytics(eventType: "trait", attributes: Record<string, unknown>)
-export function sendAnalytics(eventType: "screen", attributes: { name: string } & Record<string, unknown>)
+export function sendAnalytics(eventType: "screen", attributes?: { name: string } & Record<string, unknown>)
 export function sendAnalytics(eventType: "ui" | "track" | "operational", attributes: Record<string, unknown>)
-export function sendAnalytics(eventType: string, attributes: Record<string, unknown>): void {
-	if (!analyticsClient || !isNodeProd()){
+export function sendAnalytics(eventType: string, attributes?: Record<string, unknown>): void {
+	if (!analyticsClient && !isNodeProd()){
```

Comment:
@gxueatlassian turns out changing this to || per your suggestion in my last pr broke analytics (currently not being sent).

Reply:
Tbh it is always hard to wrap my head around multi conditions with `!`. 
Just out of curious, are we sending analytics to stage as well?

----------

Anyway, this is not a blocker, I'll leave this to your decision. Good to me. You can close this thread.

Reply:
this is a weird conditional though.  It should _always_ return if the analytics client is undefined no matter the env or else it'll create an error. Best way IMO is to do `!analyticsClient || isNodeDev()` so it never sends anything while in development and only sends it in production mode, but only if the client is available.

Reply:
This is the original code from @atrigueiro branch that got merged but happy to update.